src/main/java/com/ibanity/apis/client/products/xs2a/mappers/SynchronizationMapper.java

```diff
@@ -30,6 +31,13 @@ public static Synchronization map(DataApiModel dataApiModel) {
         if (dataApiModel.getLinks() != null) {
             result.setSelfLink(dataApiModel.getLinks().getSelf());
         }
+        if (dataApiModel.getRelationships() != null
```

Comment:
if you update src/test/resources/json/synchronization.json, you can make sure that the parsing works properly in SyncrhonizationServiceImplTest.java